appengine/pond/duck/js/duck.js

```diff
@@ -42,6 +42,15 @@ goog.require('Pond.Visualization');
  */
 Pond.Duck.editorTabs = null;
 
+/**
+ * Object holding name of tabs.
```

Comment:
This doesn't look like names, maybe reword and mention index of tab?

Reply:
done